app/assets/javascripts/common/admins.js

```diff
@@ -52,9 +40,7 @@ AdminAccess.prototype = {
 
   resourceRowCollapseListener: function () {
     const collapsibleItems = [
-      this.facilityAccessItemsPadding(),
-      this.facilityAccessItemsAccessRatio(),
-      this.facilityAccessItemsDropdown()
+      document.getElementsByClassName("access-item")
```

Comment:
This simplified how the collage handler worked so that it would respond for all levels of the hierarchy.... we don't need to grab all these elements specifically, we can use the overall access row.

Reply:
Does this change the behavior from before or it was superfluous to begin with?

Reply:
I think there is a slight change in how a row responds when you click on a checkbox, in that it will also collapse the row if you deselect it. I played with it a bit and it felt totally reasonable from a UX perspective.